Lombiq.VueJs/Lombiq.VueJs.csproj

```diff
@@ -25,14 +25,19 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Include="License.md" Pack="true" PackagePath="" />
-    <None Include="..\Readme.md" />
+    <None Include="../License.md" Link="License.md" Pack="true" PackagePath="" />
+    <None Include="../Readme.md" Link="Readme.md" />
+    <None Include="package.json" Pack="true" PackagePath="" />
     <None Include="NuGetIcon.png" Pack="true" PackagePath="" />
     <None Include="build/**" Pack="true" PackagePath="build/" />
     <None Include="*.props" Pack="true" PackagePath="" />
     <None Include="*.targets" Pack="true" PackagePath="" />
+    <None Include=".npmrc" Pack="true" PackagePath="" />
+    <None Include="Assets/**" Pack="true" PackagePath="Assets/" />
+    <None Remove="node_modules/**" />
 
-    <None Remove="node_modules\**" />
+    <!-- Necessary to avoid NU5118 due to duplicate build/Lombiq.VueJs.props files from OrchardCore.Module.Targets. -->
```

Comment:
Let's add some more details, so I don't have to go search for them 6 months from now:
```suggestion
    <!-- Necessary to avoid NU5118 due to duplicate build/Lombiq.VueJs.props files. The referenced module
         "OrchardCore.Module.Targets" tries to generate that file from the Package.Build.props mentioned below. -->
```